### ISSUE TITLE
Problem: SSH key works with `ssh -T git@github.com` however Ansible says "ssh: no key found" 

### ISSUE THREAD

**Issue by tomshomelab** [ansible-semaphore/semaphore#2511]
### Issue

Ive generated a new SSH key for accessing a private repository on github using:
`ssh-keygen -t ed25519 -C "email@gmail.com"` 

on the VM that is running semaphore and can test the key with:
`ssh -T git@github.com` 
`Hi *USERNAME*! You've successfully authenticated, but GitHub does not provide shell access.`

however when i add the same key to semaphore UI and test a task i get the following error:

```
Task 18 added to queue
Started: 18
Run TaskRunner with template: check-storage
Preparing: 18
Cloning Repository ssh://github.com/*USERNAME*/ansible
Failed updating repository: parsing private key: ssh: no key found
Running app failed: parsing private key: ssh: no key found
```

Where do i go from here, the key works but ansible doesnt see the key

### Impact

Web-Frontend (what users interact with), Ansible (task execution), Docker, Semaphore Project

### Installation method

Docker

### Database

MySQL

### Browser

Firefox, Microsoft Edge, Opera

### Semaphore Version

latest

### Logs & errors

```
Task 18 added to queue
Started: 18
Run TaskRunner with template: check-storage
Preparing: 18
Cloning Repository ssh://github.com/*USERNAME*/ansible
Failed updating repository: parsing private key: ssh: no key found
Running app failed: parsing private key: ssh: no key found
```

### Manual installation - system information

Linux docker-v3-management 5.4.0-196-generic #216-Ubuntu SMP Thu Aug 29 13:26:53 UTC 2024 x86_64 x86_64 x86_64 GNU/Linux

im using nginx reverse proxy with cloudflare

### Additional information

ive also tested mounting the ssh key location to the docker compose file but this made no difference;
```
      - /home/administrator/.ssh/id_ed25519:/root/.ssh/id_ed25519:rw
      - /home/administrator/.ssh/known_hosts:/root/.ssh/known_hosts:rw
```

**Comment by Lyuven** [comment]
Have you added the key into your key store in the Semaphore GUI AND added that Key to the repository "ansible"?
Create your project -> Key Store -> New Key -> Type: SSH key
then
Repositories-> New Repository: Name / url or path -> branch -> *Access Key* (select key created in Key Store above)

**Comment by tomshomelab** [comment]
> Have you added the key into your key store in the Semaphore GUI AND added that Key to the repository "ansible"? Create your project -> Key Store -> New Key -> Type: SSH key then Repositories-> New Repository: Name / url or path -> branch -> _Access Key_ (select key created in Key Store above)

yes i have done both these things. see image of repo added and access key added.
![image](https://github.com/user-attachments/assets/c2b65a18-97b9-4185-82b6-d864208711ab)
